pkg/api/types.go

```diff
@@ -514,6 +514,11 @@ type TestStepConfiguration struct {
 	// Postsubmit configures prowgen to generate the job as a postsubmit rather than a presubmit
 	Postsubmit bool `json:"postsubmit,omitempty"`
 
+	// Please do not use: Still under implementation.
```

Comment:
probably don't want to check this in

Reply:
Will remove it in https://github.com/openshift/ci-tools/pull/1838